views/common.twig

```diff
@@ -409,7 +410,7 @@
             var errorMessagesTrans = {
                 error: "{% trans "ERROR" %}",
                 loadingLayout: "{% trans "There was a problem loading the layout!" %}",
```

Comment:
Layout and Playlist should be capitals here (no need to amend this PR, but you could modify for the next one).

Reply:
I will create an issue for those changes and apply them on the next PR